coprocess_python.go

```diff
@@ -240,27 +240,25 @@ func PythonLoadDispatcher() error {
 // PythonNewDispatcher creates an instance of TykDispatcher.
 func PythonNewDispatcher(middlewarePath, eventHandlerPath string, bundlePaths []string) (coprocess.Dispatcher, error) {
 	CMiddlewarePath := C.CString(middlewarePath)
+	defer C.free(unsafe.Pointer(CMiddlewarePath))
 	CEventHandlerPath := C.CString(eventHandlerPath)
+	defer C.free(unsafe.Pointer(CEventHandlerPath))
 	CBundlePaths := C.CString(strings.Join(bundlePaths, ":"))
+	defer C.free(unsafe.Pointer(CBundlePaths))
```

Comment:
We had an actual leak here as there were only two frees before, correct?

Reply:
That's right, this is the missing one.